include/c2pa.h

```diff
@@ -84,6 +152,16 @@ IMPORT extern char *c2pa_version(void);
  */
 IMPORT extern char *c2pa_error(void);
 
+/**
+ * Load Settings from a string
+ * # Errors
+ * Returns -1 if there were errors, otherwise returns 0
```

Comment:
What is the formatting language for doc comments here? I'm worried that without some semantics (bullet items) this will become a run-on sentence. But I don't know if the intended doc engine will use Markdown or such.

Reply:
Applies throughout the file.

Reply:
Same for ending sentences in documentation with a period. (Throughout the PR.)

Reply:
A added periods throughout. These comments are generated from the Rust comments. I'm not sure about plans for. doc engine.